src/ol/sphere/sphere.js

```diff
@@ -58,19 +57,19 @@ ol.Sphere.prototype.cosineDistance = function(c1, c2) {
 
 
 /**
- * Returns the geodesic area for a list of coordinates.
- *
- * [Reference](http://trs-new.jpl.nasa.gov/dspace/handle/2014/40409)
- * Robert. G. Chamberlain and William H. Duquette, "Some Algorithms for
- * Polygons on a Sphere", JPL Publication 07-03, Jet Propulsion
- * Laboratory, Pasadena, CA, June 2007
- *
- * @param {Array.<ol.Coordinate>} coordinates List of coordinates of a linear
- * ring. If the ring is oriented clockwise, the area will be positive,
- * otherwise it will be negative.
- * @return {number} Area.
- * @api
- */
+* Returns the geodesic area for a list of coordinates.
+*
+* [Reference](http://trs-new.jpl.nasa.gov/dspace/handle/2014/40409)
+* Robert. G. Chamberlain and William H. Duquette, "Some Algorithms for
+* Polygons on a Sphere", JPL Publication 07-03, Jet Propulsion
+* Laboratory, Pasadena, CA, June 2007
+*
+* @param {Array.<ol.Coordinate>} coordinates List of coordinates of a linear
+* ring. If the ring is oriented clockwise, the area will be positive,
+* otherwise it will be negative.
+* @return {number} Area.
+* @api
+*/
```

Comment:
I used fixjsstyle on that file. Perhaps I had outdated rules for that that
will come good when I rebaseline.
On 22 Apr 2015 16:00, "Tim Schaub" <notifications@github.com> wrote:

> In src/ol/sphere/sphere.js
> <https://github.com/openlayers/ol3/pull/3609#discussion_r28880702>:
>
> > - * @return {number} Area.
> > - * @api
> > - */
> > +* Returns the geodesic area for a list of coordinates.
> > +*
> > +* [Reference](http://trs-new.jpl.nasa.gov/dspace/handle/2014/40409)
> > +* Robert. G. Chamberlain and William H. Duquette, "Some Algorithms for
> > +* Polygons on a Sphere", JPL Publication 07-03, Jet Propulsion
> > +* Laboratory, Pasadena, CA, June 2007
> > +*
> > +* @param {Array.<ol.Coordinate>} coordinates List of coordinates of a linear
> > +* ring. If the ring is oriented clockwise, the area will be positive,
> > +* otherwise it will be negative.
> > +* @return {number} Area.
> > +* @api
> > +*/
>
> Please don't reformat comment blocks like this. It adds unnecessary cruft
> to the patch. Comment blocks should consistently be formatted like this:
>
> /**
>  * The `*` in this line is aligned with the first `*` in the line above (not the `/`).
>  * Same goes for this line.  And the line below.
>  */
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openlayers/ol3/pull/3609/files#r28880702>.
>